src/main/java/org/lekitech/gafalag/Mapper.java

```diff
@@ -0,0 +1,114 @@
+package org.lekitech.gafalag;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.Getter;
+import lombok.SneakyThrows;
+import org.lekitech.gafalag.dictionarymodels.Article;
+import org.lekitech.gafalag.jsonmodels.Page;
+import org.lekitech.gafalag.jsonmodels.Token;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * Date: 3.11.2021
+ * Project: GafalagParser
+ * Class: Mapper
+ *
+ * @author Enver Eskendarov (envereskendarov@gmail.com)
+ * @version 1.0
+ */
+
+public class Mapper extends ObjectMapper {
+
+    private final String srcFile;
+    @Getter
+    private final List<Article> dictionary = new ArrayList<>();
+    final StringBuilder exp = new StringBuilder();
+    final StringBuilder inf = new StringBuilder();
+    final StringBuilder def = new StringBuilder();
+
+    public Mapper(String srcFile) {
+        this.srcFile = srcFile;
+    }
+
+    @SneakyThrows(value = IOException.class)
+    public Mapper init() {
+        final List<Page> pages = readerFor(new TypeReference<List<Page>>() {
+        }).readValue(new File(srcFile));
+        final List<Token> tokens = pages.stream()
+                .flatMap(page -> page.getTextBlocks().stream())
+                .map(Token::new).filter(Token::isValid).collect(Collectors.toList());
```

Comment:
Для читабельности лучше так отформатировать:
```suggestion
        final List<Token> tokens = pages.stream()
                .flatMap(page -> page.getTextBlocks().stream())
                .map(Token::new)
                .filter(Token::isValid)
                .collect(Collectors.toList());
```